src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -389,6 +390,15 @@ enum CustomDataTypeEnum {
     NUMERICAL = 'NUMERICAL',
 }
 
+interface PatientIdentifierFilter {
```

Comment:
we should put all these types in the shared/models folder.  The reason is that people end up importing them in places and it ends up bloating the webpack bundles.